src/Aspire.Hosting.Azure/AzureResourceExtensions.cs

```diff
@@ -238,4 +239,69 @@ private static void WriteSqlDatabaseToManifest(ManifestPublishingContext context
         context.Writer.WriteString("type", "azure.sql.database.v0");
         context.Writer.WriteString("parent", resource.Parent.Name);
     }
+
+    /// <summary>
+    /// Adds an Azure OpenAI resource to the application model.
+    /// </summary>
+    /// <param name="builder">The <see cref="IDistributedApplicationBuilder"/>.</param>
+    /// <param name="name">The name of the resource. This name will be used as the connection string name when referenced in a dependency.</param>
+    /// <returns>A reference to the <see cref="IResourceBuilder{AzureOpenAIResource}"/>.</returns>
+    public static IResourceBuilder<AzureOpenAIResource> AddAzureOpenAI(this IDistributedApplicationBuilder builder, string name)
```

Comment:
Can we add some Hosting tests for these changes?

Reply:
Also added one for SQL Azure, please check too